gunicorn_config.py

```diff
@@ -33,13 +37,13 @@ def json_record(
         return dict(
             namespace=settings.NAMESPACE,
             event="making request",
-            created_at=response_time.isoformat(),
+            created_at=response_time.isoformat(timespec="milliseconds") + "Z",
             data={
                 "remote_ip": record.args["h"],
-                "method": record.args["m"],
-                "path": url,
                 "status": str(record.args["s"]),
             },
+            method=record.args["m"],
+            path=path,
             severity=severity,
```

Comment:
could we have query on here as well please? and move `status` out to here a alter to `status_code`

Reply:
`query` added and `status moved to an `int` as `status_code` outside data.